db/migrate/20230203073707_create_comment.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.integer :comments_counter, default: 0
```

Comment:
- [ ] According to the ERD diagram:
![image](https://user-images.githubusercontent.com/77069011/221380577-9b0828f8-2287-4237-b761-4ba8185f92b6.png)

A comment doesn't have a `comments_counter` attribute, please consider removing it! 🤓 🔧